pkg/onit/cluster/gui.go

```diff
@@ -0,0 +1,132 @@
+// Copyright 2019-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cluster
+
+const envoyConfig = `
+## refs:
+      ## - https://www.envoyproxy.io/docs/envoy/latest/start/start#quick-start-to-run-simple-example
+      ## - https://raw.githubusercontent.com/envoyproxy/envoy/master/configs/google_com_proxy.v2.yaml
+      admin:
+        access_log_path: /tmp/admin_access.log
+        address:
+          socket_address: { address: 0.0.0.0, port_value: 9901 }
+      static_resources:
+        listeners:
+        {{- range $key, $value := .Values.onosservices }}
```

Comment:
this is based on a helm chart template - where is this value going to come from in onit?